### PR TITLE
Openpose model in container

### PR DESCRIPTION
**Pull request by CasperKjaerhus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,16 +3,26 @@
...
+then run `docker compose up -d`
 
-After docker compose has finished, nextjs should run on http://localhost and sanic should run on http://localhost:8080
+After docker compose has finished, nextjs should run on <http://localhost> and sanic should run on <http://localhost:8080>
 
 ## Docs
 
-You can access docs for the api at http://localhost:8080/docs or http://localhost:8080/docs/swagger for swaggerUI
\ No newline at end of file
+You can access docs for the api at <http://localhost:8080/docs> or <http://localhost:8080/docs/swagger> for swaggerUI
```
Comment (morten-j) [inline README.md:19]: Er det ikke port 8000?

**Review by morten-j — CHANGES_REQUESTED** [review]
smh

**Review by morten-j — APPROVED** [review]
lets go champ

**Review by mKiddeT — APPROVED** [review]
sucks to suck

**Pull request merged**